components/cdc/src/observer.rs

```diff
@@ -106,6 +181,11 @@ impl<E: KvEngine> CmdObserver<E> for CdcObserver {
     ) {
         assert!(!cmd_batches.is_empty());
         fail_point!("before_cdc_flush_apply");
+        // Untrack raw ts regardless of the ob level
```

Comment:
Nit: Suggest to add blank lines before and after this new block. To make the procedure more clear.